src/actors/symbolication.rs

```diff
@@ -47,10 +47,7 @@ use crate::utils::hex::HexValue;
 use crate::utils::sentry::SentryFutureExt as _;
 
 /// Options for demangling all symbols.
-const DEMANGLE_OPTIONS: DemangleOptions = DemangleOptions {
-    with_arguments: true,
-    format: DemangleFormat::Short,
-};
+const DEMANGLE_OPTIONS: DemangleOptions = DemangleOptions::complete();
```

Comment:
basically short + arguments would become name_only + parameters, but I think @jan-auer has a better idea what we should do here?

Reply:
Hmm, I went with `DemangleOptions::complete().return_type(false)` which seems to give the desired output because arguments are included.